.github/workflows/ci.yml

```
@@ -62,22 +62,61 @@ jobs:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download APK Artifact
+        uses: actions/download-artifact@v4
         with:
-          name: release-apk-${{ matrix.platform }}-${{ github.run_id }}
-          path: build/app/outputs/apk/release/app-release.apk
-          if-no-files-found: error
+          name: android-apk-${{ github.run_number }}
+          path: ./
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)